lua/nvim-treesitter/parsers.lua

```diff
@@ -34,6 +34,9 @@ list.javascript = {
     url = "https://github.com/tree-sitter/tree-sitter-javascript",
     files = { "src/parser.c", "src/scanner.c" },
   },
+  injections = {
+    pattern = '\\c^js$'
```

Comment:
Should we have an option instead of manually setting this? Like `injections.ignore_case`